spec/datetimeformat.html

```diff
@@ -102,16 +97,14 @@
         1. Let _opt_ be a new Record.
         1. For each row of <emu-xref href="#table-3">Table 3</emu-xref>, except the header row, do:
           1. Let _prop_ be the name given in the Property column of the row.
-          1. Let _value_ be GetOption(_options_, _prop_, *"string"*, «the strings given in the Values column of the row», *undefined*).
-          1. ReturnIfAbrupt(_value_).
+          1. Let _value_ be ? GetOption(_options_, _prop_, *"string"*, «the strings given in the Values column of the row», *undefined*).
```

Comment:
in 262, it is `*undefined*` everywhere. after a discussion with @bterlson, his recommendation for code is:

"the thing inside ` is ecmascript source text, and not a literal, e.g.: `Function.prototype`".